v0/destinations/recurly/transform.js

```diff
@@ -0,0 +1,101 @@
+const btoa = require("btoa");
+const { EventType } = require("../../../constants");
+const {
+  ACCEPT_HEADERS,
+  DEFAULT_BASE_ENDPOINT,
+  CONFIG_CATEGORIES,
+  MAPPING_CONFIG,
+  BILL_TO_SELF,
+  BILL_TO_PARENT
+} = require("./config");
+const {
+  defaultRequestConfig,
+  ErrorMessage,
+  defaultPostRequestConfig,
+  constructPayload,
+  stripTrailingSlash
+} = require("../../util");
+
+/**
+ * Backup Code If required
+ * @param {*} isSubDomain
+ * @param {*} name
+ * @returns
+ */
+const appendSubDomain = (isSubDomain, name) => {
+  return isSubDomain ? `${name}-recurly` : "";
+};
+
+const processIdentify = (message, category) => {
+  const { address } = constructPayload(
+    message,
+    MAPPING_CONFIG[CONFIG_CATEGORIES.ADDRESS.name]
+  );
+  address.postal_code = address.postal_code.toString();
+  const payload = {
+    ...constructPayload(message, MAPPING_CONFIG[category.name]),
+    address
+  };
+  payload.bill_to = BILL_TO_SELF;
```

Comment:
why are we setting this to self?

Reply:
It has two option bill_to : 'self' or 'parent'.
When u create account by default it is set to 'self'.
When an account is parent it can be set to parent.
For example- Parent account P1 has 3 child account then organization can take decision if it will be self or parent.
P1 : A1, A2, A3
A1 : 'parent'
A2 : 'parent'
A3 : 'parent'
It means all bill_to will rely on parent account to pay.
Again, I think it is something we cannot decide, but I am setting to self because its the default value.